### PR TITLE
Define API to add/remove participant's +1 to event

### PR DESCRIPTION
**Pull request by aleksey-korolev**

### CODE REVIEW

File: SignupDirector/SignupDirectorAPI/migrations/0002_participants_name_alter_participants_accompanions_and_more.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    operations = [
+        migrations.AddField(
+            model_name='participants',
+            name='name',
+            field=models.CharField(default='no_name', max_length=255),
+        ),
+        migrations.AlterField(
+            model_name='participants',
+            name='accompanions',
+            field=models.IntegerField(default=0),
```
Comment (aleksey-korolev) [inline SignupDirector/SignupDirectorAPI/migrations/0002_participants_name_alter_participants_accompanions_and_more.py:21]: there's no much reason to add a field and the alter it in the same PR

File: SignupDirector/SignupDirectorAPI/migrations/0003_participants_attendance.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    dependencies = [
+        ('SignupDirectorAPI', '0002_participants_name_alter_participants_accompanions_and_more'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='participants',
+            name='attendance',
+            field=models.BooleanField(default=False),
```
Comment (aleksey-korolev) [inline SignupDirector/SignupDirectorAPI/migrations/0003_participants_attendance.py:16]: I'd suggest to combine all migrations in the same PR into one

File: SignupDirector/SignupDirectorAPI/serializers.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+from rest_framework import serializers
+from .models import Participants
+
+class ParticipantsSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Participants
+        fields = ['id','name','attendance', 'accompanions','event_id']
```
Comment (aleksey-korolev) [inline SignupDirector/SignupDirectorAPI/serializers.py:7]: Looks like trivial serializer (db field names are the same as class field names).
Framework should allow using some default serializer without explicit class for it to make maintainability better.

File: SignupDirector/SignupDirectorAPI/migrations/0002_participants_name_alter_participants_accompanions_and_more.py (hunk 1 above)
Comment (kdylko) [reply SignupDirector/SignupDirectorAPI/migrations/0002_participants_name_alter_participants_accompanions_and_more.py:21]: done

File: SignupDirector/SignupDirectorAPI/migrations/0003_participants_attendance.py (hunk 2 above)
Comment (kdylko) [reply SignupDirector/SignupDirectorAPI/migrations/0003_participants_attendance.py:16]: done

File: SignupDirector/SignupDirectorAPI/serializers.py (hunk 3 above)
Comment (kdylko) [reply SignupDirector/SignupDirectorAPI/serializers.py:7]: I propose to leave this issue for later as I don't complete understand what kind of serializer would be better in this case. Lets discuss separately

**Pull request merged**